src/main.cpp

```diff
@@ -577,9 +580,13 @@ int main ( int argc, char** argv )
 #ifdef HEADLESS
     QCoreApplication* pApp = new QCoreApplication ( argc, argv );
 #else
+# if defined (Q_OS_IOS)
```

Comment:
I know I am picky ;-) -> Two spaces missing.

Reply:
Should be solved